native/core/src/events/rsvp.rs

```diff
@@ -7,12 +7,18 @@ use strum::Display;
 use super::BelongsTo;
 
 /// RSVP status
+// previolusly accepted only camel-case
+// now accept not only serialization in kebab-case but also alias in camel-case
+// because a lot of RsvpStatus in camel-case were uploaded and if parsing under only kebab-case failed at RsvpStatus in camel-case
```

Comment:
```suggestion
// previously accepted only PascalCase
// now accept will serialize to kebab-case but also deserialize the previously
// posted PascalCase variants
```


(starting uppercase letter isn't [camelCase, but PascalCase](https://www.freecodecamp.org/news/snake-case-vs-camel-case-vs-pascal-case-vs-kebab-case-whats-the-difference/) ...)

Reply:
Thank you for your correction.